app/models/admin_auction_status_presenter_factory.rb

```diff
@@ -60,8 +60,10 @@ def other_purchase_card_presenter
       AdminAuctionStatusPresenter::PendingAcceptance
     elsif auction.accepted_pending_payment_url?
       AdminAuctionStatusPresenter::AcceptedPendingPaymentUrl
-    elsif auction.accepted?
-      AdminAuctionStatusPresenter::Accepted
+    elsif auction.accepted? && auction.paid_at.nil?
+      AdminAuctionStatusPresenter::OtherPcard::Accepted
```

Comment:
Nice! added an issue for also namespacing the other presenters https://github.com/18F/micropurchase/issues/1317